lxd/storage/drivers/driver_powerflex_utils.go

```diff
@@ -727,6 +774,22 @@ func (d *powerflex) getHostNQN() string {
 	return fmt.Sprintf("nqn.2014-08.org.nvmexpress:uuid:%s", d.state.ServerUUID)
 }
 
+// getHostGUID returns the SDC GUID.
+// The GUID is unique for a single host.
+// Cache the GUID as it never changes for a single host.
+func (d *powerflex) getHostGUID() (string, error) {
+	if d.sdcGUID == "" {
+		guid, err := goscaleio.DrvCfgQueryGUID()
+		if err != nil {
+			return "", fmt.Errorf("Failed to query SDC GUID: %w", err)
+		}
+
+		d.sdcGUID = strings.TrimSuffix(guid, "\n")
```

Comment:
Concerning that the goscaleio package isnt doing that already.

Reply:
Would using https://pkg.go.dev/strings#TrimSpace be a safer option here?

Reply:
Mhhh thanks, that is a left over as I previously ran the SDC binary which had a newline in it's output. I haven't removed those bits when switching to the `goscaleio` library.

Fixed.